devel/git-crypt/Portfile

```diff
@@ -51,4 +51,6 @@ build.env               CXX=${configure.cxx} \
                         "LDFLAGS=${configure.ldflags}  [get_canonical_archflags ld]" \
                         PREFIX=${prefix}
 
-destroot.args           ENABLE_MAN=yes PREFIX=${prefix}
+destroot.args           ENABLE_MAN=yes \
+                        PREFIX=${prefix} \
+                        DOCBOOK_XSL=${prefix}/share/xsl/docbook-xsl-nons/manpages/docbook.xsl
```

Comment:
Why is it necessary to specify the filesystem path this way? docbook-xsl is normally used by referencing one of the stylesheet's canonical web URLs, and the docbook-xsl software takes care of mapping those known URLs to MacPorts filesystem paths so that no network access actually occurs.

Reply:
had to specify `XML_CATALOG_FILES=${prefix}/etc/xml/catalog`, because `depends_build-append    bin:xsltproc:libxslt` doesn't install libxslt and uses `/usr/bin/xsltproc`.

please advise a better way to handle this.